src/components/AdminPortal/Venues/Modal/VenueModal.module.css

```
@@ -1,22 +1,31 @@
-  box-shadow: 0 var(--shadow-offset-sm) var(--shadow-blur-sm)
-    var(--color-shadow-light);
+  border: var(--border-width-sm) solid var(--input-shadow);
+  box-shadow: var(--shadow-sm);
+}
+
+.previewVenueModal {
+  display: flex;
+  position: relative;
+  width: 100%;
+  overflow: hidden;
+  /* Ensures content doesn't overflow the card */
```

Stylelint (17.2.0)

[error] Expected empty line before comment (comment-empty-line-before)

(comment-empty-line-before)